ports/stm32f3/Makefile

```diff
@@ -0,0 +1,79 @@
+UF2_FAMILY_ID = 0x57755a57
+CROSS_COMPILE = arm-none-eabi-
+
+ST_HAL_DRIVER = lib/st/stm32f3xx_hal_driver
+ST_CMSIS = lib/st/cmsis_device_f3
+CMSIS_5 = lib/CMSIS_5
+
+# List of git submodules that is included as part of the UF2 version
+GIT_SUBMODULES = CMSIS_5 st tinyusb
+
+include ../make.mk
+
+# Port Compiler Flags
+CFLAGS += \
+  -flto \
+  -mthumb \
+  -mabi=aapcs \
+  -mcpu=cortex-m4 \
+  -mfloat-abi=hard \
+  -mfpu=fpv4-sp-d16 \
+  -nostdlib -nostartfiles \
+  -DCFG_TUSB_MCU=OPT_MCU_STM32F3 \
+	-Wno-uninitialized
+
+# suppress warning caused by vendor mcu driver
+CFLAGS += -Wno-error=cast-align -Wno-error=unused-parameter
+
+LD_FILES ?= $(PORT_DIR)/linker/STM32F303VCTx_FLASH.ld
+
+# Port source
+PORT_SRC_C += \
+	$(addprefix $(CURRENT_PATH)/, $(wildcard *.c)) \
+	$(ST_CMSIS)/Source/Templates/system_stm32f3xx.c \
+	$(ST_HAL_DRIVER)/Src/stm32f3xx_hal.c \
+	$(ST_HAL_DRIVER)/Src/stm32f3xx_hal_cortex.c \
+	$(ST_HAL_DRIVER)/Src/stm32f3xx_hal_rcc.c \
+	$(ST_HAL_DRIVER)/Src/stm32f3xx_hal_gpio.c \
+	$(ST_HAL_DRIVER)/Src/stm32f3xx_hal_flash.c \
+	$(ST_HAL_DRIVER)/Src/stm32f3xx_hal_flash_ex.c \
+	$(ST_HAL_DRIVER)/Src/stm32f3xx_hal_uart.c
+
+SRC_C += \
+	$(PORT_SRC_C) \
+	lib/tinyusb/src/portable/st/synopsys/dcd_synopsys.c \
```

Comment:
stm32f303 use different dcd than f4 change this to `lib/tinyusb/src/portable/st/stm32_fsdev/dcd_stm32_fsdev.c`